app/models/manageiq/providers/google/cloud_manager/refresh_parser.rb

```diff
@@ -5,13 +5,13 @@ module Google
     class CloudManager::RefreshParser < ManageIQ::Providers::CloudManager::RefreshParser
       include Vmdb::Logging
 
-      def self.ems_inv_to_hashes(ems, options = nil)
-        new(ems, options).ems_inv_to_hashes
+      def self.ems_inv_to_hashes(target, inventory, options = nil)
```

Comment:
I don't think you need target here.

Take a look at [the vmware `refresh_parser`](https://github.com/ManageIQ/manageiq/blob/778c9df3b7382047caf0678ef1e1c2594dded9f3/app/models/manageiq/providers/vmware/infra_manager/refresh_parser.rb#L10).  By the time it gets to the parser, it doesn't even really care about the `target` anymore.